evap/staff/forms.py

```diff
@@ -133,7 +134,7 @@ def clean(self):
     def save(self, *args, **kwargs):
         course_type = super().save(*args, **kwargs)
         if "name_en" in self.changed_data or "name_de" in self.changed_data:
-            update_template_cache(Evaluation.objects.filter(state="published", course__type=course_type))
+            update_evaluation_template_cache(Evaluation.objects.filter(state="published", course__type=course_type))
```

Comment:
because of the overview list on each evaluation detail page that includes the course's information

Reply:
i don't understand, does the evaluation line in that overview list change when changing the course's type?

Reply:
ah, i think i confused the evaluation line and evaluation page here. you're right, then only the course template cache should need to be updated.
only if the course name changes and the evaluation has no name it will have an effect on the evaluation line as well.